README.md

```diff
@@ -173,6 +173,10 @@ EVM emulation and testing is hard. Echidna has a number of limitations in the la
 | Lack of support for function pointers in Solidity | [#798](https://github.com/crytic/echidna/issues/798) | *on hold* |
 ## Installation
 
+### Building using Homebrew (MacOS)
+
+You can install Echidna with `brew install echidna`.
+
```

Comment:
Thanks for the PR! I think that other title goes a bit better indeed. It'd also be cool to highlight this also installs the extra dependencies like Slither, and it can be used to make a source build.

```suggestion
### Homebrew (macOS / Linux)

If you have Homebrew installed on your Mac or Linux machine, you can install Echidna and all of its dependencies (Slither, crytic-compile) by running `brew install echidna`.

You can also compile and install the latest `master` branch code by running `brew install --HEAD echidna`

You can get further information in the [`echidna` Homebrew Formula](https://formulae.brew.sh/formula/echidna) page. The formula itself is maintained as part of the [homebrew-core repository](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/echidna.rb)

```

Reply:
I appreciate the feedback! This might be a little bit stupid question but shall I click the "commit suggestion" button to incorporate the proposed changes to the pull request? I haven't had the opportunity to work on a large project yet 😅

Reply:
Yep, that's one way to do it! You can also incorporate the changes locally and commit them as you usually would, then push the branch again to github.